Doc/library/zipfile.rst

```diff
@@ -551,11 +559,19 @@ file:
    If *arcname* is not specified, the name will be the same as *filename*, but
    with any drive letter and leading path separators removed.
 
+   The *strict_timestamps* keyword argument, when set to ``False``, allows to
```

Comment:
I don't think that it's needed to repeat here that it's a *keyword* argument, just write "argument".

Reply:
I think that it's better to mention it here too, a user reading the docs because of this specific function might not notice the other function with the same argument.